designer/server/src/main/scala/pl/touk/nussknacker/ui/api/TapirCodecs.scala

```diff
@@ -88,4 +131,23 @@ object TapirCodecs {
 
   }
 
+  object ProcessingModeCodec {
+    implicit val processingModeSchema: Schema[ProcessingMode] = Schema.string
+  }
+
+  object EngineSetupNameCodec {
+    implicit val engineSetupNameSchema: Schema[EngineSetupName] = Schema.string
+  }
+
+  object ProcessNameCodec {
+    implicit val processNameSchema: Schema[ProcessName] = Schema.string
+  }
+
+  object ScenarioWithDetailsForMigrationsCodec {
+
+    import ProcessNameCodec._
+
+    implicit val scenarioWithDetailsForMigrationsSchema: Schema[ScenarioWithDetailsForMigrations] = Schema.anyObject
```

Comment:
I wrote it in another comment I would like to relax it ATM and left it as TODO. Typing this is no trivial since
there are a lot of substructures and some of them are not Tapir-schema friendly (e.g. `TypingResult`)

Reply:
ok, let's add the proper `// TODO:`

Reply:
I'm not sure if for the determined developer, the comment will be clear. Maybe we could add the link to this discussion?